users/user-manager.js

```diff
@@ -32,33 +32,33 @@ var createUser = (json) => {
   return new response.ResponseObject(201, {"uid":"1", "url":"//www.google.com"});
```

Comment:
Put a TODO here to indicate that this needs to be hooked up to the `user-manager`